js/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- Node.JS 4,5, or 6+
```

Comment:
Node 6 will be the "current" LTS in 8 days according to:
https://github.com/nodejs/LTS#lts-schedule .  This will probably coincide with Node 7 release.  Once that happens 4 will probably roll off the homepage.